frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentAccordionBody/EnvironmentAccordionBody.tsx

```diff
@@ -77,6 +78,13 @@ const StyledAlert = styled(Alert)(({ theme }) => ({
     marginInline: theme.spacing(2), // should consider finding a variable here
 }));
 
+const StyledStrategySeparator = styled(StrategySeparator)(({ theme }) => ({
+    [`&:has(+ *:not(ol) .${disabledStrategyClassName}), &:has(+ ol > li:first-of-type .${disabledStrategyClassName})`]:
```

Comment:
Yeah, this one's kinda nasty. I've been coming around to the suggestion that we don't need to nest the strategy list if you have milestones, which would make this into the more manageable `&:has(+ * .${disabledStrategyClassName})`. But I'll do that in a separate PR.